docs/visualizations/README.md

```diff
@@ -0,0 +1,35 @@
+# Create visualizations
```

Comment:
Mention that the files in this folder (e.g., OpenStudio-BuildStock/docs/visualizations/dependencyWheels/dep_wheel.html) are blank templates so users aren't confused as to why they are blank. Would be good to mention that in the template text itself too.

Reply:
I added a commit to make this more explicit. There is now language in the README.md and the file has been renamed.